nixos/modules/misc/ids.nix

```diff
@@ -346,6 +346,7 @@ in
       zoneminder = 314;
       paperless = 315;
       #mailman = 316;  # removed 2019-08-30
+      pixiecore = 317;
```

Comment:
I don't think we need a statically allocated uid/gid here.
The service itself has no state to maintain.

```suggestion
```